### ISSUE TITLE
Proposal : What is really Haxe Trailer !

### ISSUE THREAD

**Issue by delahee** [HaxeFoundation/Project-Management#44]
Hi,

Now is a turning tide, haxe is gaining traction fast, everybody hear of it but there is a definetly a gap between hearsay and adoption. We have tons of video presentations about tech points but we are really missing the manager stage because..managers don't watch tech presentations that much...
I also had some multiple reports that Haxe does not break through management stages because they do not grasp what is haxe and how its easy to make it fit one business needs.

I suggest we put up with trailers, like flashy presentations made by true motion designers, that clearly shows What is haxe, what not etc. this should take the form of dynamic youtube videos or something.

If there are some already done, let's centralize here !

**Comment by ianharrigan** [comment]
+1

**Comment by MatthijsKamstra** [comment]
+1

**Comment by andyli** [comment]
I'm not sure about "a programming language" trailer. But I think a video showcase of all the good games (or whatever) made in Haxe would be awesome!

**Comment by delahee** [comment]
@andyli 
Sorry to clarify/disagree but that is exactly not what I propose

TLDR : I think the next battle will be explaining exactly what is haxe and how it fits for people who only have limited time to grant to yet another new technology and not big companies with large R&D teams that can dig that swiftly. It's already easy to cinvince haxe is used by big names, let's show more.

In the long :

We discussed with Ian and the whole point was about how hard it is to actually grasp what IS Haxe actually to suggest it could be part of workflows. 

Most people won't understand what is a transpiler, how it manipulates code, what impact it does have, why all apis are not then magically available where it plugs and how it can patch workflows to inject more productivity.

Many people will not dig informations as scattered as they are so our marketing duty is to present the information in a comprehensible form.

Off course we can show how big companies use Haxe, but I tend to think that is battle is "nearly" already won, with big names like tivo or massive for sony or great comm groups like disney, viacom, tf1 etc...we can now demonstrate that haxe is viable. But...

But first explaining efficiently what is haxe in a didactic (pedagogic) way could help :)

I think we can build up a prezi around this idea swift enough then post it on youtube after proofreadingby people active in this topic ^^

What do you think ?

**Comment by ianharrigan** [comment]
Yeah, a sort of quick video presentation, that doesnt get _too_ deep into details, but adequately shows haxe "moving" to other languages, utilizing existing code bases (in those languages) to create apps / games, etc. 

The basic idea would be a short video that management and budget types can watch to get a general overview of what haxe is and how it could potentially fit in to their company. In fact, even a diagram of that type of thing seems to be missing from haxe.org.

**Comment by andyli** [comment]
Ummm.. I think I get what you mean. I usually think that it is better done by having a nice front page and giving talks in conferences. You want to express it in the form of a trailer, which may be a good idea.
It would be great if you can have a quick prototype of that, because it is hard for me to imagine for now : )

For the video showcase of made of haxe, it is my need from my own experience presenting haxe in conferences. It takes a lot of time to describe them in a talk - you know time is always not enough to cover all the great things. And I think video trailer is a nature medium of showcasing games/apps/etc.
But yeah, a trailer on Haxe the language itself could also save time in an intro-to-haxe talk. So let's just have something quick prototype to discuss further.

**Comment by delahee** [comment]
ok then @andyli can you open a separate issue about a haxe video showcase, I'll come up with something as to "What is Haxe" video, I'll prezi something early december unless someone propose something first.

**Comment by Merelleya** [comment]
Hey @delahee are you still looking into this?

**Comment by delahee** [comment]
Nope, the silex team has not answered, propably for their own reasons ;).  They started a video section so I guess they sort of have an agenda. 

As to me, in december I could not work on anything serious due to personnal reasons.I strictly don't have time and priority on this these days, so if you feel about closing, proceed :)

**Comment by lexoyo** [comment]
I don't remember you asked something?
Are you looking for a teaser video we did?
https://www.youtube.com/watch?v=XBMJYpxoP70
https://www.youtube.com/watch?v=9bjs3mq0jVQ
https://www.youtube.com/watch?v=KU425Zrzxyk
https://www.youtube.com/watch?v=UxloLXZiOXI

**Comment by delahee** [comment]
@lexoyo yep I sent a mail on 06/11/2015 , i'll send it again :) 

edit : sent, keep me posted ;)

**Comment by delahee** [comment]
Lack of time, I close.
